nulib/meadow
# Works.add_representative_image/1 sets placeholders by work type when representative_file_set is nil

# Summary 
Calling `representative_image` on a work without a representative file set will choose a placeholder based on the work type.

# Specific Changes in this PR
- `Works.add_representative_image/1`
- `Meadow.Indexing.V2.Work.representative_file_set/1` sets the `aspect_ratio` to `1.0` when a representative file set is not present 
# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
Add a work without a file set and check the `representative_image` field e.g. `https://iiif.dev.rdc.library.northwestern.edu/iiif/2/[YOUR DEV ENV]/00000000-0000-0000-0000-000000000002` for A/V works and  `https://iiif.dev.rdc.library.northwestern.edu/iiif/2/[YOUR DEV ENV]/00000000-0000-0000-0000-000000000001` for Image works

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
